package.json

```diff
@@ -109,7 +109,7 @@
     "@elastic/ems-client": "7.11.0",
     "@elastic/eui": "30.2.0",
     "@elastic/filesaver": "1.1.2",
-    "@elastic/good": "8.1.1-kibana2",
+    "@elastic/good": "watson/good#fix-pipeline",
```

Comment:
Let's get this merged upstream before merging this so that we don't have a git dependency and the slowness that comes along with them in master.

Reply:
Yeah I agree. I opened the PR up to reviews to get that started, but will not merge the PR until I've updated this dependency to a released version

Reply:
Fixed